opencensus/trace/internal/local_span_store_impl.cc

```diff
@@ -127,7 +127,7 @@ std::vector<SpanData> LocalSpanStoreImpl::GetLatencySampledSpans(
         latency_ns < filter.upper_latency_ns) {
       out.emplace_back(span);
     }
-    if (out.size() >= filter.max_spans_to_return) break;
+    if (out.size() >= static_cast<size_t>(filter.max_spans_to_return)) break;
```

Comment:
Please don't do this. These sign conversion warnings are fixed by https://github.com/census-instrumentation/opencensus-cpp/pull/11

Reply:
Done